src/dedicated/overview.md

```diff
@@ -0,0 +1,15 @@
+# Platform.sh Dedicated
+
+Platform.sh Dedicated is a robust, redundant layer on top of Platform.sh Professional.  It is well-suited for those who like the Platform.sh development experience but need more resources and redundancy for their production environment.  It is available only with an Enterprise contract.
+
+Platform.sh Dedicated consists of two parts: The Development Environment and the Dedicated Cluster.
+
+### The Development Environment
+
+The Development Environment is a normal [Platform.sh Professional](https://docs.platform.sh/) account, with all of the capabilities and workflows of Platform.sh Professional.  The one difference is that the `master` branch will not be associated with a domain and thus will never be "production".  The [Platform.sh Professional Documentation](https://docs.platform.sh/) has more information on how to best leverage it.
```

Comment:
This link here seems old to me now since Ent/Professional docs won't have a distinction anymore. Where should it actually be directed?